src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -520,9 +522,21 @@ Namespace Microsoft.CodeAnalysis.Semantics
         End Function
 
         Private Function CreateBoundConversionOperation(boundConversion As BoundConversion) As IConversionExpression
-            Dim operand As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundConversion.Operand))
+            Dim operand As Lazy(Of IOperation)
+            Dim methodSymbol As MethodSymbol
+
+            ' TODO: Use Gen's GetConversion after https://github.com/dotnet/roslyn/pull/21445 is merged.
```

Comment:
>' TODO: Use Gen's GetConversion after https://github.com/dotnet/roslyn/pull/21445 is merged. [](start = 12, length = 92)

I think our current policy is to use ```PROTOTYPE(<feature>)``` instead of "TODO:". Alternatively you can open an issue to track the work and remove the TODO.

Reply:
Opened https://github.com/dotnet/roslyn/issues/21797, as I'm not sure when #21445 will be merged.